src/main/java/digital/slovensko/autogram/ui/gui/GUI.java

```diff
@@ -212,6 +208,34 @@ public void showError(AutogramException e) {
         stage.show();
     }
 
+    public void showErrorPortInUse(Stage stage, Autogram autogram, int portNumber, AutogramException e) {
```

Comment:
Tuto vieme vyuzit asi len `showError` a novy exception kde zabalime co treba.

Reply:
Nakoniec sme sa dohodli, že nie showError, ale použijeme dizajn toho warningu, akurát možno na to vyrbím všeobecný view, aby nemal každý warning wvoj vlastný controller a view.